README.md

```diff
@@ -118,12 +118,7 @@ you need to have the following firmware files:
 `/lib/firmware/contr_lm.bin`  
 `/lib/firmware/Progmodel_*.bin`  
 
-You can retrieve these from:
```

Comment:
Please leave this list of option for finding firmware. The firmware files indicated in lantiq_fw are only useful to a small subset of people, and other people may find their firmwares using these instructions.